unpackaged/config/rd2_post_config/layouts/npe03__Recurring_Donation__c-Enhanced Recurring Donations Layout.layout

```diff
@@ -24,6 +24,10 @@
                 <behavior>Required</behavior>
                 <field>npe03__Amount__c</field>
             </layoutItems>
+            <layoutItems>
+                <behavior>Readonly</behavior>
```

Comment:
@force2b , please let me know if the new fields display need to be changed.
@rponti-sforg , the RD2 layout is deployed and viewable after RD2 is enabled and metadata deployed either via the link or `cci task run deploy_rd2_config`.

Reply:
Thanks @RadSFDeveloper. The fields looks correct (readonly), though I didn't actually verify their position on the page.

Reply:
I think this is good, though I have to admit that I did not see the "Next Year Value" field initially. I wonder if these should be in the right column of the Schedule Details section, possibly below "Number of Paid Installments". @sam-knox What do you think?
<img width="656" alt="image" src="https://user-images.githubusercontent.com/801017/73463434-3b04cd00-434b-11ea-80f4-041fbca3c71d.png">

Reply:
I replied on the WI, but my suggestion was to just swap Current Year with Campaign. That way the lookups are together on the left and Current and Next year are together on the right.

I actually wouldn't mind addressing the page layout holistically in a separate work item. For example, I don't think that Payment Method should be where it is. I think Effective Date is a little buried too.

It might make sense to create a new section called "Recurring Donation Statistics" that contains Current Year, Next Year, Total Paid, and Number of Paid Installments.

If we can incorporate user feedback, we should do that too.